.github/workflows/release.yml

```diff
@@ -0,0 +1,23 @@
+name: CD-release
```

Comment:
Should the name here just be "Release" ?

Reply:
Oh, good catch, I'll amend